doc/source/whatsnew/v0.20.0.txt

```diff
@@ -99,6 +99,34 @@ support for bz2 compression in the python 2 c-engine improved (:issue:`14874`).
 
 .. _whatsnew_0200.enhancements.uint64_support:
```

Comment:
this tag should go after this section. Then add a similar type tag for this.

``.. _whatsnew_0200.enhancements.pickle_compression:`` or something.